prow/config/config.go

```diff
@@ -1194,6 +1211,14 @@ func (c *Config) validateComponentConfig() error {
 		if _, err := url.Parse(v); err != nil {
 			return fmt.Errorf(`Invalid value for Planks job_url_prefix_config["%s"]: %v`, k, err)
 		}
+		// TODO(@sbueringer): Remove in September 2020
+		if strings.HasSuffix(v, "gcs/") {
+			logrus.Warning(strings.Join([]string{
+				"configuring the 'gcs/' storage provider suffix in the job url prefix is now deprecated, ",
+				"please configure the job url prefix without the suffix as it's now appended automatically. Handling of the old ",
+				"configuration will be removed in September 2020",
```

Comment:
Removing support for the old URL format would break existing links which doesn't seem reasonable to me. Why would we need to stop supporting the old format in order to support the new one?

Reply:
Its only about configuration and supporting configuration with /gcs. We don't want to stop handling /gcs requests